.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: go
+
+go:
+  - '1.12.x'
+
+services:
+  - docker
+
+env:
+  global:
+    - GO111MODULE=on
+
+go_import_path: github.com/bblfsh/performance
+
+install:
+  - go mod download
+  - go generate ./...
+
+script:
+  - go test -v ./...
```

Comment:
Do we also want to include benchmarks?

Reply:
nope, I added this line just in case some unit tests to appear, now this command is not necessary